app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> None:
+    split_command = command.split()
+    if split_command[1] == split_command[2]:
+        return
+
+    if split_command[0] == "cp":
+        with (
+            open(split_command[1], "r") as file_input,
+            open(split_command[2], "w") as file_output
+        ):
+            file_output.write(file_input.read())
```

Comment:
The function assumes that the source file exists and can be opened for reading, and that the destination file can be opened for writing. If this is not the case, the function will raise an exception. Consider adding some error handling to make your code more robust. For example, you can catch the `FileNotFoundError` and `PermissionError` exceptions and return an error message.